content/docs/use-cases/ci-cd-ml.md

```diff
@@ -0,0 +1,70 @@
+# Continuous Integration and Deployment for Machine Learning
+
+MLOps and DataOps apply DevOps methodologies to machine learning (ML) and data
+management. The fields are rapidly evolving beyond a mere set of best practices,
+and now encompass the entire product lifecycle management. A core component is
+the automation of testing and deployment: CI/CD (continuous integration and
+delivery).
+
+CI/CD is easy to set up with practically any Git repository. Some example
+scenarios are:
+
+- Agile development
+  - Automating quality assurance (QA) means teams can confidently deploy new
+    versions several times a day and even before the weekend without fear of
+    bugs/regressions
+- Easy testing
+  - When new data is pushed in a pull request (PR), validation tests
+    automatically run in the cloud (without each collaborator setting up
+    potentially inconsistent local testing environments)
+  - When a new model is pushed in a PR, reports (on performance against the
+    current deployed models) are automatically generated and posted as a PR
+    comment with graphs and tables
+- Continuous machine learning (CML): online training
+  - Schedule regular jobs pulling in data updates from public sources to refine
+    deployed models
+  - Provision bare metal in the cloud and run a brute force hyperparameter
+    search
+
+In ML, CI/CD can be used to automatically build, test and push data and models
+to production in the cloud with minimal effort. There are many components, such
+as:
+
+1. Data collection & storage
+2. Metadata management
+3. Validation & testing (including provisioning resources)
+4. Sharing & deployment
+5. Monitoring & feedback
```

Comment:
could perhaps be replaced with some sort of a diagram along these lines <sub>(pun intended)</sub>:

![image](https://user-images.githubusercontent.com/10780059/115491420-dbe4ee00-a257-11eb-81cd-537e03449fcf.png)

Reply:
Can you explain a bit what these components and diagram try to convey? What I see is where DVC/CML fit into 
 or expand established CI/CD. Is that the idea?

Reply:
p.s. from here on is where @dberenbaum can probably help more than I.

Reply:
Love the idea of the diagram if it can be easily explained. Maybe better for a second iteration?

Reply:
have a new sketchy (pun intended) diagram: ![](https://static.iterative.ai/img/ml-vs-cicd.png)

things to improve:
- instead of "ML" (top) and "CML" (bottom) labels, use "Traditional Machine Learning (ML)" and "CI/CD for ML"
- increase spacing between top & bottom parts

Reply:
I like the circularity of the bottom half.

Reply:
Original intention: The top half of the fig is really a "Key" while the bottom half is the real fig.

However after discussing with @shcheklein and thinking about the target audience; I think we're going to actually keep only the top half of the fig here. The bottom half can be moved down to the DVC/CML section.